### PR TITLE
Delete disassembly data for ntsc-1.2

### PR DESCRIPTION
**Pull request by cadmic**
The disassembly data and the unknown syms check are annoying to maintain and not needed now that https://github.com/cadmic/oot/tree/ntsc-1.2 matches ntsc-1.2, I recommended building that and copying it into expected/ instead.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -26,12 +26,11 @@ pipeline {
...
             }
         }
-        stage('Build ntsc-1.2, check disasm metadata') {
+        stage('Build ntsc-1.2') {
             steps {
                 sh 'ln -s /usr/local/etc/roms/oot-ntsc-1.2-us.z64 baseroms/ntsc-1.2/baserom.z64'
                 sh 'make -j$(nproc) setup VERSION=ntsc-1.2'
                 sh 'make -j$(nproc) VERSION=ntsc-1.2'
-                sh '.venv/bin/python3 tools/check_disasm_metadata_unksyms.py -v ntsc-1.2'
                 sh 'make clean assetclean VERSION=ntsc-1.2'
```
Comment (fig02) [inline Jenkinsfile:34]: shouldnt we wait until this matches upstream to do this?

Comment (cadmic) [reply Jenkinsfile:34]: well I don't personally use the disassembly anymore now that there's a way to get a matching build. But you guys seem to be doing a lot of renames so I figured this doesn't pass the cost/benefit test

Comment (fig02) [reply Jenkinsfile:34]: I just mean the jenkins build specifically. should we comment the task out or something until its fully matching and it has a purpose to run?

Comment (cadmic) [reply Jenkinsfile:34]: oh I see. Right now this checks that the ntsc-1.2 build isn't broken at least which is kind of nice but maybe not worth the extra build time

Comment (fig02) [reply Jenkinsfile:34]: thats fine to me, i thought it had 0 value rn

**Review by fig02 — APPROVED** [review]

**Comment by Dragorn421** [comment]
I would prefer to only delete it when the version matches

**Comment by fig02** [comment]
now extra relevant with 1.2 matching in main. needs to merge upstream to resolve conflicts

**Comment by fig02** [comment]
I will likely merge when ci is done even if it doesnt have a second review, this is pretty straight forward

**Pull request merged**